babel/src/main/java/org/apache/calcite/sql/babel/SqlDateTimeAtLocal.java

```diff
@@ -48,7 +59,8 @@ public SqlDateTimeAtLocal(SqlParserPos pos, SqlNode dateTimePrimary) {
     return ImmutableNullableList.of(dateTimePrimary);
   }
 
-  @Override public void unparse(SqlWriter writer, int leftPrec, int rightPrec) {
+  @Override public void unparse(SqlWriter writer, int leftPrec,
```

Comment:
nit: Could remove the newline, I think this would fall under 80 chars

Reply:
It seems to be at 81 chars if the new line is removed.